sources/vega/oracles/v1/data.proto

```diff
@@ -13,9 +14,11 @@ message OracleData {
   // data holds all the properties of the oracle data
   repeated Property data = 2;
   // matched_specs_ids lists all the oracle specs that matched this oracle data.
+  // When it's left empty, it means no oracle spec matched this oracle data.
   repeated string matched_spec_ids = 3;
-  // broadcast_at is the time at which the data was broadcast for the first
-  // time.
+  // broadcast_at is the time at which the data was broadcast to the markets
```

Comment:
Perhaps we could be more precise, is time in milliseconds or nanoseconds? (Im guessing the latter based on how we represent time elsewhere)